Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM maven:3.6-jdk-13
```

Comment:
Given the LTS is 11 and 17, and JDK 11 is already retired https://togithub.com/docker-library/openjdk/issues/505
I made the update to use version 11 which probably has minimal impact of the case.

Reply:
makes sense - might just have to check with deployment if it uses the same parameters for enabling low pause GC

Reply:
@inodb sounds good. Thanks! Feel free to merge after testing.